kinder/pkg/cluster/status/node.go

```diff
@@ -296,30 +289,18 @@ func (n *Node) WriteNodeSettings(settings *NodeSettings) error {
 		return errors.Wrapf(err, "failed to encode %s", nodeSettingsPath)
 	}
 
-	var lastError error
 	dir := filepath.Dir(nodeSettingsPath)
-	attempts := 0
```

Comment:
the poll here on directory creation / file write is being removed from this function because polling is now being done after node creation.